log/log_test.go

```diff
@@ -64,113 +66,100 @@ func TestSingleton(t *testing.T) {
 
 func TestConstructionNil(t *testing.T) {
 	t.Parallel()
-	stats, _ := statsd.NewNoopClient(nil)
-	_, err := NewAuditLogger(nil, stats, stdoutLevel)
+	_, err := New(nil, stdoutLevel)
 	test.AssertError(t, err, "Nil shouldn't be permitted.")
 }
 
 func TestEmit(t *testing.T) {
 	t.Parallel()
 	log := setup(t)
 
-	log.AuditNotice("test message")
+	log.AuditInfo("test message")
 }
 
 func TestEmitEmpty(t *testing.T) {
 	t.Parallel()
 	log := setup(t)
 
-	log.AuditNotice("")
+	log.AuditInfo("")
 }
 
-func ExampleAuditLogger() {
-	audit := setup(nil)
-
-	audit.clk = clock.NewFake()
-	audit.AuditErr(errors.New("Error Audit"))
-	audit.WarningErr(errors.New("Warning Audit"))
-	// Output: [31m00:00:00 log.test ERR [AUDIT] Error Audit[0m
-	// [33m00:00:00 log.test WARNING Warning Audit[0m
-}
-
-func TestEmitErrors(t *testing.T) {
-	t.Parallel()
-	audit := setup(t)
+func ExampleLogger() {
+	impl := setup(nil)
 
-	audit.AuditErr(errors.New("Error Audit"))
-	audit.WarningErr(errors.New("Warning Audit"))
+	bw, ok := impl.w.(*bothWriter)
+	if !ok {
+		fmt.Printf("Wrong type of impl's writer: %T\n", impl.w)
+		return
+	}
+	bw.clk = clock.NewFake()
+	impl.AuditErr(errors.New("Error Audit"))
+	impl.Warning("Warning Audit")
+	// Output: [31m[1mE000000 log.test [AUDIT] Error Audit[0m
```

Comment:
I think you can start the output on the next line,  right? That would look better.